phpunit

```diff
@@ -11,7 +11,7 @@
  */
 
 // Please update when phpunit needs to be reinstalled with fresh deps:
-// Cache-Id-Version: 2016-03-23 14:50 UTC
+// Cache-Id-Version: 2016-03-25 09:44 UTC
```

Comment:
@xabbuh can you please rebase your PR on latest 2.7 (with 2.3 just merged) and put here the same value as in 2.3? (I did not merge the cache-id change in 2.7 to not break it, this PR will, once tests are green).